tests/update_only_fields/tests.py

```diff
@@ -256,6 +257,16 @@ def post_save_receiver(**kwargs):
         pre_save.disconnect(pre_save_receiver)
         post_save.disconnect(post_save_receiver)
 
+    def test_empty_update_fields_positional(self):
```

Comment:
While I don't object to this new test, I'm not sure what's the goal of adding it. Is there any logic/code path that is covered in this test but in the other tests?

Reply:
The test fails on `main` because empty `update_fields` is falsey - the second issue I listed on the ticket.

```
$ ./runtests.py --parallel 1 update_only_fields
Testing against Django installed in '/Users/chainz/Documents/Projects/django/django'
Found 17 test(s).
Creating test database for alias 'default'...
System check identified no issues (0 silenced).
.F...............
======================================================================
FAIL: test_empty_update_fields_positional (update_only_fields.tests.UpdateOnlyFieldsTests.test_empty_update_fields_positional)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/chainz/Documents/Projects/django/tests/update_only_fields/tests.py", line 264, in test_empty_update_fields_positional
    with (
  File "/Users/chainz/Documents/Projects/django/django/test/testcases.py", line 128, in __exit__
    self.test_case.assertEqual(
AssertionError: 1 != 0 : 1 queries executed, 0 expected
Captured queries were:
1. UPDATE "update_only_fields_person" SET "name" = 'Sara', "gender" = 'F', "pid" = NULL WHERE "update_only_fields_person"."id" = 1

----------------------------------------------------------------------
```

Reply:
(This regression was the reason I marked the ticket as a release blocker when I saw it.)

Reply:
Thank you both! I have added the `async` version for this issue :rocket: